Allow cloning to new destination account (#10)
silinternational/terraform-enterprise-migrator: README.md

2. If you're a Go developer you can install it by running `go get -u https://github.com/silinternationa/terraform-enterprise-migrator`
3. If you're a Go developer and want to modify the source before running, clone this repo and run with `go run main.go ...`

## Cloning a TF Cloud Workspace
Examples.

Get help about the command.

```$ go run main.go clone -h```

Clone just the workspace (no variables) to the same organization.

> ```$ go run main.go clone -o=my-org -s=source-workspace n=new-workspace```

Reviewer: ```suggestion
```$ go run main.go clone -o=my-org -s=source-workspace -n=new-workspace```
```

Clone a workspace and its variables (but not its state) to a different organization in TF Cloud.

> Note: Sensitive variables will get a place holder in the new workspace, the value of

Reviewer: ```suggestion
Note: Sensitive variables will get a placeholder in the new workspace, the value of
```

which will need to be corrected manually.  Also, the environment variables from the source
workspace will need to be moved in the destination workspace from the normal variables section
down to the environment variables section (e.g. `CONFIRM_DESTROY`).

```
$ go run main.go clone -c=true -o=org1 -p=org2 -d=true \
$   -s=source-workspace -n=destination-workspace -v=org2-vcs-token
```

Note: To copy over the state from one workspace to a clone, see lib/V2Client.RunTFInit
  for an idea of how you would do that manually using the `terraform init` command. 

>   Remember, it depends on the ALTAS_TOKEN environment variable to be set correctly first

Reviewer: ```suggestion
  Remember, it depends on the ATLAS_TOKEN environment variable to be set correctly first
```
